Open_Source/Section-5/5.8_DragonFly.md

```diff
@@ -0,0 +1,93 @@
+# Dragonfly
+
+<p align="center">
+  <img src="https://repository-images.githubusercontent.com/309874357/c2b3ff80-a931-11eb-89b2-32c6ceed11cd" />
+</p>
+
+<br>
+
+# What is Dragonfly
+
+<p>
+  Dragonfly is an intelligent P2P-based image and file distribution tool. It aims to improve the efficiency and success rate of file transferring, and maximize the usage of network bandwidth, especially for the distribution of larget amounts of data, such as application distribution, cache distribution, log distribution, and image distribution.
+ At Alibaba, every month Dragonfly is invoked two billion times and distributes 3.4PB of data. Dragonfly has become one of the most important pieces of infrastructure at Alibaba.
+
+  While container technologies makes DevOps life easier most of the time, it surely brings some challenges: for example the efficiency of image distribution, especially when you     have to replicate image distribution on several hosts.
+  
+  The goal of Dragonfly is to tackle distribution problems in cloud native scenarios.
+  The project is comprised of three main components:
+   - supernode plays the role of central scheduler and controls all distribution procedure among the peer network 
+   - dfget resides on each peer as an agent to download file pieces 
+   - and “dfdaemon” plays the role of proxy which intercepts image downloading requests from container engine to dfget 
+</p>
+
+# Main Features
+<br>
+<p> This project is an open-source version of the Dragonfly used at Alibaba. You can find it <a href="https://github.com/dragonflyoss/Dragonfly" target="_blank" rel="noreferrer noopener"> here </a> </p>
+
+<h2> Main Dragonfly Features: </h2>
+<p>
+  - <strong>P2P-based file distribution</strong>: By using the P2P technology for file transmission, it makes the most out of the bandwidth resources of each peer to improve downloading efficiency, and saves a lot of cross-IDC bandwidth, especially the costly cross-board bandwidth.
```

Comment:
<img width="1020" alt="Screenshot 2021-09-19 at 2 17 11 PM" src="https://user-images.githubusercontent.com/44888949/133921269-c507ebe1-c3a4-4320-bad6-e2dda9c72ec0.png">
Bullet isn't appearing for this. Can you please fix it?

Reply:
Done ! Check it Now